zipline/data/resample.py

```diff
@@ -450,3 +453,48 @@ def volumes(self, assets, dt):
                     volumes.append(val)
                     continue
         return np.array(volumes)
+
+
+class MinuteResampleSessionBarReader(SessionBarReader):
+
+    def __init__(self, calendar, minute_bar_reader):
+        self._calendar = calendar
+        self._minute_bar_reader = minute_bar_reader
+
+    def _get_resampled(self, columns, start_dt, end_dt, assets):
+        minute_data = self._minute_bar_reader.load_raw_arrays(
```

Comment:
```
Signature: zipline.data.minute_bars.MinuteBarReader.load_raw_arrays(self, fields, start_dt, end_dt, sids)
Docstring:
Retrieve the arrays of pricing data for the given coordinates of
``fields`` (OHLCV), minute range [``start_dt``, ``end_dt``] and sids.

Parameters
----------
fields : iterable of str
    The OHLCV fields ('open', 'high', 'low', 'close', 'volume') for
    which to read data.
start_dt : pd.Timestamp
    The first minute of the date range for which to read data.
end_dt : pd.Timestamp
    The last minute of the date range for which to read data.
sids : iterable of int
    The sid identifiers for which to retrieve data.

Returns
-------
raw_arrays : list of ndarray
    A list where each item corresponds with the fields in the order
    the fields are given.
    Each item is a 2D array with a shape of (minutes_in_range, sids)
    The OHLC arrays are floats; the 'volume' array is ints.
```

Already handled as instantiation of the `minute_bar_reader`, though adding a calendar argument may be a way to not use the `CME` minutes for all futures when using `load_raw_arrays`.